### PR TITLE
Healing Hands 2 -- ball scratching edition

### PR DESCRIPTION
**Pull request by ghost**
basically touch people to heal 5 brute+burn damage. doesn't work on androids/synths/borgs/yourself. 7.5 second cooldown.

it works but it needs sprites

🆑 More Robust Than You
add: Healing hands mutation
/🆑

### CODE REVIEW

**Comment by Iamgoofball** [comment]
>try to make it not easy to instantly heal people no effort
>you powercreep genetics

File: code/datums/mutations.dm [hunk 1]
```diff
@@ -646,6 +646,60 @@ GLOBAL_LIST_EMPTY(mutations_list)
 	if(owner.a_intent == INTENT_HARM)
 		owner.LaserEyes(target)
 
+/datum/mutation/human/healing_hands
```
Comment (kevinz000) [inline code/datums/mutations.dm]: Please make this clear that this will automatically overwrite an existing mutation if the DNA block count is not raised.

Comment (MoreRobustThanYou) [reply code/datums/mutations.dm]: ok

Comment (MoreRobustThanYou) [reply code/datums/mutations.dm]: gimme a minute

File: code/modules/mob/living/carbon/human/human_defines.dm [hunk 2]
```diff
@@ -43,6 +43,10 @@
 
 	var/drunkenness = 0 //Overall drunkenness - check handle_alcohol() in life.dm for effects
 	var/datum/personal_crafting/handcrafting
+
+	var/healaura = FALSE
+	var/mutable_appearance/healaura_overlay
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/human/human_defines.dm]: nope, fuck this.

File: code/datums/mutations.dm [hunk 3]
```diff
@@ -646,6 +646,60 @@ GLOBAL_LIST_EMPTY(mutations_list)
...
+	name = "Healing Hands"
+	quality = POSITIVE
+	get_chance = 27.5
+	lowest_value = 256 * 12
+	text_gain_indication = "<span class='notice'>You feel a comforting buzzing in your hands.</span>"
+	time_coeff = 5
+	var/next_heal_time = 0
```
Comment (Cruix) [inline code/datums/mutations.dm]: Mutation datums are singletons, so everyone who has the mutation uses the same datum instance. If you do it this way, everyone who has the mutation will share the same cooldown. If you want everyone to have their own cooldown, you will have to store it in a var on the mob or in an associative list, with the mobs as keys, on the datum.

Comment (MoreRobustThanYou) [reply code/datums/mutations.dm]: Fixing that rn

File: code/modules/mob/living/carbon/human/human_defines.dm (hunk 2 above)
Comment (MoreRobustThanYou) [reply code/modules/mob/living/carbon/human/human_defines.dm]: ok removed

**Comment by KorPhaeron** [comment]
No thank you goof is right

**Pull request closed without merging**